index.md

```diff
@@ -415,14 +415,13 @@ Window Azure
   </div>
 </section>
 <section class="content-contact-us" id="ABOUT">
-<div>
-<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1937.2262589149777!2d100.5643517064789!3d13.81184444759192!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x30e29c4fecf9c121%3A0xd211864fd071fd2b!2sSalween+Solutions+Co.%2C+Ltd.!5e0!3m2!1sth!2sth!4v1494321073791" width="1900" height="600" frameborder="0" style="border:0" allowfullscreen></iframe>
+<div id="google-map">
 </div>
 </section>
 
 <section>
   <div>
-    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1937.2262589149777!2d100.5643517064789!3d13.81184444759192!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x30e29c4fecf9c121%3A0xd211864fd071fd2b!2sSalween+Solutions+Co.%2C+Ltd.!5e0!3m2!1sth!2sth!4v1494321073791" width="1900" height="600" frameborder="0" style="border:0" allowfullscreen></iframe>
+    "id = google-map" Google map Area
```

Comment:
Ah OK, it's just for reference while working on the design. We'll tidy it away tomorrow. @pektoon :eyeglasses: